mangaki/mangaki/views.py

```diff
@@ -807,6 +809,90 @@ def legal_mentions(request):
     return render(request, 'mangaki/legal.html')
 
 
+def fix_index(request):
+    suggestion_list = Suggestion.objects.select_related('work', 'user').prefetch_related(
+        'work__category', 'evidence_set__user').all().order_by('is_checked', '-date')
+
+    paginator = Paginator(suggestion_list, FIXES_PER_PAGE)
+    page = request.GET.get('page')
+
+    try:
+        suggestions = paginator.page(page)
+    except PageNotAnInteger:
+        suggestions = paginator.page(1)
+    except EmptyPage:
+        suggestions = paginator.page(paginator.num_pages)
+
+    context = {
+        'suggestions': suggestions
+    }
+
+    return render(request, 'fix/fix_index.html', context)
+
+
+def fix_suggestion(request, suggestion_id):
+    cluster_colors = {
+        'unprocessed': 'text-info',
+        'accepted': 'text-success',
+        'rejected': 'text-danger'
+    }
+
+    if not suggestion_id:
+        return redirect('fix-index')
+
+    suggestion = get_object_or_404(Suggestion.objects.select_related('work', 'user'), id=suggestion_id)
+
+    evidence = None
+    if request.user.is_authenticated:
+        evidence = Evidence.objects.filter(user=request.user, suggestion=suggestion).first()
```

Comment:
Do we have any reason to not use `get` and enforce the uniqueness of Evidence per `(user, suggestion)` ? (for the non-existence, we can catch the DoesNotExist exception and set `evidence` to `None` or whatever).